index.html

```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
                 </div>
             </div>
         </section>
-    </main>
+    </section>
+
+    <section class="contact-section">
+        <h1 id="contact-card-title">Contact</h1>
+
+        <p>If you have an application you are interested in developing, a feature that you need
+            built or a project that needs coding. I’d love to help with it </p>
+
+        <form action="#" method="POST" class="contact-form"></form>
+        <input type="text" name="_name" placeholder="Name" class="form-field" required>
```

Comment:
- [ ] Please ensure to add the attribute `maxlength = "30" ` for the name input tag. This is so as to add and improve on the validations for your form.